pkg/service/volume_service_test.go

```diff
@@ -318,6 +324,37 @@ func (suite *VolumeServiceTestSuite) TestACLVolume() {
 	}
 }
 
+func (suite *VolumeServiceTestSuite) TestExpiredToken() {
+
+	volume := suite.createVolume("TestExpiredToken")
+	if volume != nil {
+		// Get the token expiry time
+		sToken := suite.groupService.client.SessionToken
```

Comment:
Could we do something simple like this to avoid the long wait?

suite.groupService.client.SessionToken = "expired"

Reply:
Yes we can do that . thanks 

`func (suite *VolumeServiceTestSuite) TestExpiredToken() {

	volume := suite.createVolume("TestExpiredToken")
	if volume != nil {
		//Set invalid session Token 9d255b36c700ec8b56e2064e67f01c45
		suite.groupService.client.SessionToken = "9d255b36c700ec8b56e2064e67f01c45"
		suite.deleteVolume("TestExpiredToken")
	}
}`